.github/workflows/test.yml

```diff
@@ -27,23 +27,30 @@ jobs:
         python:
           - version: "3.7"
             toxenv: py37
+            experimental: false
```

Comment:
Would GitHub Actions work if we just skip defining these for falsy values to avoid them in each single job? Now they show up like this:

![image](https://user-images.githubusercontent.com/16777978/167309021-033d0308-89e2-493a-9816-a7844868d507.png)

Or alternatively define it at top-level only once via `matrix.experimental` and then only add an include where this is positive? Like they do here https://michaelheap.com/testing-experimental-versions-github-actions/#testing-multiple-versions?

I haven't tried it myself so not sure would need to play with this a bit.